bin/diagnostics.bash

```diff
@@ -42,37 +47,54 @@ while [ "${#}" -gt 0 ] ; do
             command_arg="${2:-}"
             shift
             ;;
-        -h | --help)
-            usage
-            ;;
         *)
-            log_error "error: invalid argument: \"${1:-}\""
+            log_error "ERROR: invalid argument: \"${1:-}\""
             usage
             ;;
     esac
     shift
 done
 
-if [ -z "${CK8S_PGP_FP:-}" ]; then
-    fingerprints=$(yq4 '.creation_rules[].pgp' "${sops_config}")
+log_self_managed_notice() {
+    log_warning "WARNING: Notice for self-managed customers:"
 
-    log_warning "Notice for self-managed customers:"
-    echo -e "\tEncrypting using the fingerprints: $fingerprints." 1>&2
-    echo -e "\tIf you want to send diagnostic data to Elastisys, make sure to do:\n" 1>&2
+    echo -e "\tIf you want to send diagnostic data to Elastisys, make sure to store GPG keys" 1>&2
+    echo -e "\tretrieved during onboarding in a file named:\n" 1>&2
```

Comment:
This is not appropriate for this repository since it is open source. We should not assume that Elastisys is owning the platform.

Reply:
The notice was added as when the script was first created, see the discussion [here](https://github.com/elastisys/compliantkubernetes-apps/pull/2043#discussion_r1523258042). Maybe @cristiklein can chip in regarding this?

Reply:
I agree with @simonklb that this is one of the few instance when we talk with "company voice" as opposed to talking with "project voice". As far as I saw from other project, it is okay to advertise support for an open source project. Hence, I propose the following:

```
If you are an Elastisys self-managed customer, you can send diagnostic data to Elastisys.
Make sure to store GPG keys retrieved during onboarding in a file named:
```